ui/pages/confirmation/confirmation.js

```diff
@@ -215,6 +222,7 @@ export default function ConfirmationPage({
     ApprovalType.SnapDialogAlert,
     ApprovalType.SnapDialogConfirmation,
     ApprovalType.SnapDialogPrompt,
+    ...Object.values(SNAP_MANAGE_ACCOUNTS_CONFIRMATION_TYPES),
```

Comment:
Note to self: This needs to be wrapped but the build flags dont play well with the nested comments.